src/test/credentials/provider/ssoAccessTokenProvider.test.ts

```diff
@@ -51,26 +51,32 @@ describe('SsoAccessTokenProvider', () => {
         interval: 1,
     }
 
-    function setUpStubCache(returnRegistration?: SsoClientRegistration, returnAccessToken?: SsoAccessToken) {
-        if (returnAccessToken) {
-            sandbox.stub(cache, 'loadAccessToken').returns(returnAccessToken)
-        } else {
-            sandbox.stub(cache, 'loadAccessToken').returns(undefined)
-        }
-
+    function setUpStubCache(returnRegistration?: SsoClientRegistration) {
         if (returnRegistration) {
             sandbox.stub(cache, 'loadClientRegistration').returns(returnRegistration)
         } else {
             sandbox.stub(cache, 'loadClientRegistration').returns(undefined)
         }
 
+        sandbox.stub(cache, 'loadAccessToken').returns(undefined)
+
         sandbox.stub(cache, 'saveClientRegistration').returns()
     }
 
+    let clock: lolex.InstalledClock
+
+    before(() => {
+        clock = lolex.install()
+    })
+
     afterEach(async () => {
```

Comment:
Add a `clock.reset()` to the `afterEach`

Reply:
Added into afterEach block